### ISSUE TITLE
Iptables error, docker networks lost, containers only created

### ISSUE THREAD

**Issue by jhg** [ASIRLasGalletas/Docker-Aula-Masso#1]
![spectacle m24040](https://user-images.githubusercontent.com/1288711/32421011-7337bb8a-c28b-11e7-9eb7-359338811096.png)

**Comment by DanielCleanDev** [comment]
Esto es cuando ejecutas el init.sh? A mi no me pasa eso.
Revisare igualmente.

**Comment by jhg** [comment]
Error in the `docker network create $1` command:
```
Error response from daemon: Failed to Setup IP tables: Unable to enable SKIP DNAT rule:  (iptables failed: iptables --wait -t nat -I DOCKER -i br-00560f3c7f17 -j RETURN: iptables: No chain/target/match by that name.
```

**Comment by jhg** [comment]
Si @DanielPKS eso es al ejecutar el `init.sh`, el primero es cuando esta activado el firewall, algunas de las reglas por defecto de IpTables en Suse seguramente restringe lo suficiente para que luego Docker no pueda hacer algunas de las ordenes pasadas, pero con eso desactivado sigue habiendo algún problema al crear las redes.

**Comment by DanielCleanDev** [comment]
Puedes mostrarme como tienes las iptables (iptables -nL si no estoy mal) y como quedaron las redes de docker (docker network ls - docker network inspect nombredelared).

A ver pero en mi maquina no daba error debe ser alguna configuracion en tu maquina.

**Comment by jhg** [comment]
Exacto, pero no la configuración, no se como me las apañe que conseguí corromper la instalación de Docker entera 😅 pero ya hoy consegui reinstalar Docker, aproveche para actualizar todo el sistema y parece ahora va bien.
